raymon/profiling/extractors/structured/element.py

```diff
@@ -41,7 +41,7 @@ def to_jcr(self):
 
     @classmethod
     def from_jcr(cls, jcr):
-        return cls(**jcr)
+        return cls(**jcr["state"])
```

Comment:
No, this should remain as it is. It is in your tests, that call this method, that you need to extract the state before calling this function.